samples/nrf_desktop/src/modules/config.c

```diff
@@ -58,7 +58,8 @@ static int settings_set(int argc, char **argv, void *val_ctx)
 	if (!loaded_data[id]) {
 		loaded_data[id] = new_config_event();
 		loaded_data[id]->id = id;
-		loaded_data[id]->store_needed = false;
+		/* Do not store nor forward loaded config to other devices */
+		loaded_data[id]->recipient = 0;
```

Comment:
naming this member recipient may be misguiding

Reply:
Normally recipient holds 0x52DE (mouse) or 0x52DC (dongle). I removed redundant `store_needed` member and used `recipient = 0` as a value showing that config shouldn't be forwarded, that no one is the recipient. I can use a named define for this 0 or bring back `store_needed` next to `recipient` if you prefer.